src/helpers/getSep8NextStepOnSuccess.ts

```diff
@@ -9,28 +9,54 @@ import { Sep8ApprovalStatus, Sep8Step } from "types/types.d";
 export const getSep8NextStepOnSuccess = ({
   currentStep,
   approvalStatus,
+  didUndergoKyc,
 }: {
   currentStep: Sep8Step;
   approvalStatus?: Sep8ApprovalStatus;
+  didUndergoKyc?: boolean;
 }): Sep8Step => {
   const nextStepDict: { [key in Sep8Step]: Sep8Step } = {
     [Sep8Step.DISABLED]: Sep8Step.STARTING,
-    [Sep8Step.STARTING]: (() => {
-      const approvalStatusDict: { [key in Sep8ApprovalStatus]: Sep8Step } = {
-        [Sep8ApprovalStatus.ACTION_REQUIRED]: Sep8Step.ACTION_REQUIRED,
-        [Sep8ApprovalStatus.PENDING]: Sep8Step.PENDING,
-        [Sep8ApprovalStatus.REVISED]: Sep8Step.TRANSACTION_REVISED,
-        [Sep8ApprovalStatus.SUCCESS]: Sep8Step.TRANSACTION_REVISED,
-        [Sep8ApprovalStatus.REJECTED]: currentStep,
-      };
-      return approvalStatusDict[approvalStatus!];
-    })(),
+    [Sep8Step.STARTING]: nextStepAfterApprovalServer({
+      currentStep,
+      approvalStatus,
+    }),
     [Sep8Step.PENDING]: Sep8Step.DISABLED,
     [Sep8Step.TRANSACTION_REVISED]: Sep8Step.COMPLETE,
     [Sep8Step.ACTION_REQUIRED]: Sep8Step.SENT_ACTION_REQUIRED_FIELDS,
-    [Sep8Step.SENT_ACTION_REQUIRED_FIELDS]: Sep8Step.STARTING,
+    [Sep8Step.SENT_ACTION_REQUIRED_FIELDS]: didUndergoKyc
+      ? nextStepAfterApprovalServer({
+          currentStep,
+          approvalStatus,
+        })
+      : Sep8Step.STARTING,
     [Sep8Step.COMPLETE]: Sep8Step.DISABLED,
   };
 
+  console.log(currentStep, "->", nextStepDict[currentStep]);
```

Comment:
Do we need to `console.log` here?

Reply:
Thanks for spotting that: https://github.com/stellar/stellar-demo-wallet/pull/198/commits/422e5899ec5cc4b6e58d46397e6e3e9b61a11f59